### ISSUE TITLE
Code issues

### ISSUE THREAD

**Issue by wx24598** [ChenhongyiYang/QueryDet-PyTorch#49]
Hello author, your job is very good. Can you answer the meaning of the following code for me?

        for i in range(-1*self.context, self.context+1):
            for j in range(-1*self.context, self.context+1):
                sparse_ys.append(ys+i)
                sparse_xs.append(xs+j)

        sparse_ys = torch.cat(sparse_ys, dim=0)
        sparse_xs = torch.cat(sparse_xs, dim=0)

        good_idx = (sparse_ys >= 0) & (sparse_ys < fh) & (sparse_xs >= 0)  & (sparse_xs < fw)
